app/models/uncut.rb

```diff
@@ -0,0 +1,49 @@
+class Uncut < MediaResource
+  after_save :cut_contents
+
+  serialize :segmentation, JSON
+
+  validates :medium, inclusion: {in: %w[audio]}, if: :status_complete?
+  validate :validate_segmentation
+
+  def cut_contents
+    if status_complete? && segmentation.present? && segmentation_previously_changed?
+      # TODO
```

Comment:
Will tackle this in a separate PR, for #642.  Once the user sets the `segmentation`, we need to fire off N porter jobs to cut up the file into `Content`s.